nucypher/cli/commands/stake.py

```diff
@@ -60,6 +60,15 @@
 option_worker_address = click.option('--worker-address', help="Address to assign as an Ursula-Worker", type=EIP55_CHECKSUM_ADDRESS)
 
 
+def _setup_emitter(general_config):
+    # Banner
+    emitter = general_config.emitter
+    emitter.clear()
```

Comment:
What about removing this? I know it's highly subjective and it doesn't come from this PR, but I don't like the command to be so opinionated wrt to the console output, plus, in the end my console ends up with a lot of wasted space when I need to scroll up.

Reply:
Out of scope, IMO.